### ISSUE TITLE
Make logging calories easier

### ISSUE THREAD

**Issue by rolandgeider** [wger-project/wger#575]
Not everybody has a nutritional plan, some people just want to log the daily calories. The GUI should be more accomodating to this.

**Comment by nopinter** [comment]
@derekli17 and I would like to take this issue!

**Comment by rolandgeider** [comment]
it's yours

**Comment by nopinter** [comment]
We are "unclaiming" this issue to work on 581 instead!

**Comment by klu17** [comment]
@rolandgeider What would this entail?

**Comment by rolandgeider** [comment]
I'm not completly sure to be honest. I was thinking of adding a new preference and if that is set, we would hide the links to create plans and make the diary entry more prominent. In the end, this is more a UX question

**Comment by Dieterbe** [comment]
i wonder how a daily estimate of just a kcal number can be remotely accurate enough to be useful. at the end of the day, do they still remember what they ate all day?  perhaps more useful is to predefine "fake meals" where people can enter a certain kcal and macros. so they can at least refer to that repeatedly if they eat the same on multiple days (or multiple times a day), and this would kind of "integrate" with our current system.  but this needs input from people who actually want something like this.

**Comment by Baggypants** [comment]
At the risk of being reductionist. "Make it like myfitnesspal"(Or Waistline https://github.com/davidhealey/waistline)  where I just have a list of food from off of what I ate, at predefined mealtimes. Breakfast, Lunch, Dinner, Snacks. I'm completly lost with the planner as it is.

**Comment by Dieterbe** [comment]
@Baggypants are you saying you want to define your daily meals (using OFF ingredients), and then the app will know automatically that's what you eat every day ? 
What we have now is similar, in that you can define all your daily meals, but you need to log them explicitly every day. (This comes with some advantages like being able to define multiple meals and choosing which you ate, you can skip meals, replace them , etc )

**Comment by Baggypants** [comment]
No, I mean It seems like I have to create a nutritional plan everyday, and I need to put in the goals each time.

**Comment by Dieterbe** [comment]
No that's not how wger works. Maybe you're using an old, buggy app where the meals would disappear. Now it should make more sense.
